core/templates/domain/exploration/InteractionObjectFactory.ts

```diff
@@ -486,7 +486,7 @@ export class InteractionObjectFactory {
 
   createAnswerGroupsFromBackendDict(
       answerGroupBackendDicts: readonly AnswerGroupBackendDict[],
-      interactionId: string | null
+      interactionId: string,
```

Comment:
I think you don't need the comma at the end, we don't really do that elsewhere and it wasn't there before.

Reply:
backend representation of interactionId is null when we create a default state before actual state is saved.

Reply:
> I think you don't need the comma at the end, we don't really do that elsewhere and it wasn't there before.

done.